packages/test/test-end-to-end-tests/src/test/gc/gcTombstoneDataStores.spec.ts

```diff
@@ -791,13 +809,13 @@ describeNoCompat("GC data store tombstone tests", (getTestObjectProvider) => {
 				clientType: "interactive",
 			},
 			{
-				eventName: "fluid:telemetry:FluidDataStoreContext:GC_Tombstone_DataStore_Changed",
-				callSite: "process",
```

Comment:
Why delete the callsite in these?

Reply:
Because the call site isn't logged anymore from within GCTelemetry